test/groovy/InfraStepTests.groovy

```diff
@@ -38,6 +38,8 @@ class InfraStepTests extends BaseTest {
     script.withDockerCredentials() {
       isOK = true
     }
+    print "as it been executed within the dockers credentials ? "
```

Comment:
```suggestion
    print "Has it been executed within the Docker credentials?"
```

Reply:
👍 no this should be deleted ... sorry a debug leftover ;)